snuba/datasets/configuration/generic_metrics_v2/entities/gauges.yaml

```diff
@@ -0,0 +1,161 @@
+version: v1
+kind: entity
+name: generic_metrics_gauges_v2
+
+schema:
+  [
+    { name: org_id, type: UInt, args: { size: 64 } },
+    { name: project_id, type: UInt, args: { size: 64 } },
+    { name: metric_mri, type: String },
+    { name: timestamp, type: DateTime },
+    { name: bucketed_time, type: DateTime },
+    { name: granularity, type: UInt, args: { size: 16 } },
+    { name: use_case_id, type: String },
+    {
+      name: tags,
+      type: Nested,
+      args:
+        {
+          subcolumns:
+            [
+              { name: key, type: String },
+              { name: value, type: String },
+            ],
+        },
+    },
+    {
+      name: min,
+      type: AggregateFunction,
+      args: { func: min, arg_types: [{ type: Float, args: { size: 64 } }] },
+    },
+    {
+      name: max,
+      type: AggregateFunction,
+      args: { func: max, arg_types: [{ type: Float, args: { size: 64 } }] },
+    },
+    {
+      name: sum,
+      type: AggregateFunction,
+      args: { func: sum, arg_types: [{ type: Float, args: { size: 64 } }] },
+    },
+    {
+      name: count,
+      type: AggregateFunction,
+      args: { func: count, arg_types: [{ type: UInt, args: { size: 64 } }] },
```

Comment:
Same thing here, this was a typo. Should be

`{ func: sum, arg_types: [{ type: UInt, args: { size: 64 } }] }`

since the aggregate count would be the sum of the individual counts sent by individual gauges.